package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "github-nvim-theme",
-  "version": "0.0.7",
```

Comment:
I went ahead and just removed the version here. I did this because including it might add needless (maintenance) overhead as it would need to be updated prior to each release. It shouldn't be a problem since this repo isn't an npm package, but let me know if you'd like me to put it back in (and at the correct value).

Reply:
It's fine to exclude it.